apps/site/lib/site_web/controllers/stop_controller.ex

```diff
@@ -66,19 +66,26 @@ defmodule SiteWeb.StopController do
         |> Repo.get()
 
       if stop do
-        routes_by_stop = Routes.Repo.by_stop(stop_id, include: "stop.connecting_stops")
-
-        conn
-        |> assign(:breadcrumbs, breadcrumbs(stop, routes_by_stop))
-        |> meta_description(stop, routes_by_stop)
-        |> render("show-redesign.html", %{
-          stop_id: stop_id,
-          routes_by_stop: routes_by_stop
-        })
+        if Repo.has_parent?(stop) do
+          conn
+          |> redirect(to: stop_path(conn, :show, Repo.get_parent(stop)))
+          |> halt()
+        else
+          routes_by_stop = Routes.Repo.by_stop(stop_id, include: "stop.connecting_stops")
+
+          conn
+          |> assign(:breadcrumbs, breadcrumbs(stop, routes_by_stop))
+          |> meta_description(stop, routes_by_stop)
+          |> render("show-redesign.html", %{
+            stop_id: stop_id,
+            routes_by_stop: routes_by_stop
```

Comment:
nit-pick(non-blocking): This can be addressed in later clean up, but I don't think we use the `routes_by_stop` property on the template page (`show-redesign.html`).